android/java/org/chromium/chrome/browser/app/BraveActivity.java

```diff
@@ -560,6 +561,60 @@ public void finishNativeInitialization() {
                         BraveVpnUtils.SUBSCRIPTION_PARAM_TEXT, getPackageName());
             }
         }
+        if (OnboardingPrefManager.getInstance().isDormantUsersEngagementEnabled()
+                || getPackageName().equals(BRAVE_PRODUCTION_PACKAGE_NAME)) {
+            OnboardingPrefManager.getInstance().setDormantUsersNotificationTime(
+                    RetentionNotificationUtil.DORMANT_USERS_DAY_14, setTimeInMillis(14 * 24 * 60));
```

Comment:
is this correct? 14 * 24 * 60 is correct # of seconds; but for milliseconds it needs to be `* 1000`
(same with below)

Reply:
@bsclifton that's correct as we pass value to setTimeInmiilis in minutes and then in that function we need to convert it in millis for notifications.

Reply:
ah ok - makes sense; `setTimeInMillis` is adding that many minutes 👍